yoga cherries

markgoddard opened the pull request:
[xena] fec5b5a1c9 None feat: automatic update of community files stackhpc/xena

  SKIP: created by stackhpc-ci bot

[xena] 1e63dd9aae If7d6e58b19f98ccb7cc4c209e458cb6f4f4765ad Add support for Rocky Linux 8

  SKIP: present on yoga

[xena] a29f10e16a I4857e170ad75d2824701df583d1312b3398b93f4 Update CentOS Stream base image

  SKIP: present on yoga

[xena] 2f16e80cfd Ia8c927c330b9428a3824a6925f6274cbc54314a0 Support elements repositories for overcloud host images

  SKIP: present on yoga

[xena] 5b5df80be7 I366fbe98d27fa70b1aeb398c129f626fe042b5df Support building multiple disk images

  SKIP: present on yoga

[xena] ae2e0ed2fc I7ca7608d36910eb519340641d037dc6601ab5fb2 Fix up unit tests for proxy backport

  KEEP: Use original commit

[xena] 9e89264957 I6424fc405894514a63fb2b641637bbb9d5c070c0 Fix no_proxy configuration

  SKIP: present on yoga

[xena] 90f987fbed I6515d028914e8956eac23662c3714a908fda6ac4 proxy: add ansible issue 8743 workaround

  SKIP: present on yoga

[xena] bec34228ad I37b9d18f0523c121357c5a37ec6fc458209f8e79 Add support for root filesystem UUID customisation

  SKIP: present on yoga

[xena] 769ca36738 Ic5130a7512d4a26354bd292b0ab51ab4a9279f0a Add support for configuring proxy settings

  SKIP: present on yoga

[xena] d5fb949025 I2823016294e7df63f63be9ab26535b3962a71ebe Add support for dellos10 switch configuration

  CHOICE: present on multiple branches

    [master] 1fbb5cb400

[xena] 969bfefd57 I319dd51ed3221826f820fbc0ae3639b89e9c82ea Adds kolla_ansible_venv_ansible

  SKIP: present on yoga

[xena] 6142e31cba Id3972c24022aeb6421494c3cccdc8e7cbce802e6 libvirt: support SASL authentication

  SKIP: present on yoga

[xena] 559716e579 I73fef63fb886a9d543d2f4231fb009523495edb3 libvirt: deploy libvirt on the host

  SKIP: present on yoga

[xena] 82a288f221 I2f6c2f92903815973865ef0f5d6b867d5b995bd5 Add overcloud_dib_upper_constraints_file variable

  SKIP: present on yoga

[xena] c8e26fe32a I350f58c8a82f0f31608b34054e804c5c198d6806 Support for untemplated dirs in kolla config

  SKIP: present on yoga

[xena] a93fe43d87 I2c2378ebd99ac02586518e80e4b86a9d765476e6 Add dependencies for EFI and LVM based overcloud images

  SKIP: present on yoga

[xena] aa98c9c1e5 I93d242889e225b4e60254f6b9cc5eeb457294ac8 Build overcloud host image directly with DIB

  SKIP: present on yoga

[xena] 04133e3142 None feat: automatic update of workflows stackhpc/xena

  SKIP: created by stackhpc-ci bot

Proposed cherries:
git cherry-pick -x 1fbb5cb400  # Add support for dellos10 switch configuration
\# git cherry-pick -x ae2e0ed2fc  # SKIPPED (no longer required) Fix up unit tests for proxy backport

dougszumski commented on ansible/roles/kolla-openstack/tasks/config.yml:
```diff
@@ -41,6 +42,15 @@
+- name: Ensure extra fluentd filter configuration files exist
+  template:
+    src: "{{ item }}"
+    dest: "{{ kolla_node_custom_config_path }}/fluentd/filter/{{ item | basename }}"
```
If someone added a filter, ran kayobe, removed it, and then re-ran kayobe, would the original filter persist here? In that case should we clear the folder before syncing the filters?

oneswig replied:
Do we know that this is the only source of filters though - how would we know the filters we've removed from the filters others have added?  Do any filters come packaged in the Kolla fluentd container for example?

dougszumski replied:
I was thinking about kayobe-config - hopefully everyone should check their filters into there in a timely fashion? That should address the first point. Kolla fluentd does provide filters, but those shouldn't be in the kayobe-config?

markgoddard replied:
It's a valid point. In this case this should be the only source of filters, as the destination is the kolla-ansible config on the ansible control host (localhost), not the container config.

We can't really remove all the files, as that would break idempotence. I guess we could check for files that are in the destination and not the source.

dougszumski commented:
I just tested this and it works for me.

jovial committed:
Add support for dellos10 switch configuration

Change-Id: I2823016294e7df63f63be9ab26535b3962a71ebe
(cherry picked from commit 1fbb5cb4001e85469e60c392f47e5cca5e4e3c62)

markgoddard committed:
Support additional build host packages for IPA & overcloud

Adds support for installing additional build host dependencies when
building IPA and overcloud host images via
'ipa_build_dib_host_packages_extra' and
'overcloud_dib_host_packages_extra'.

Support for installing additional host packages was added in
stackhpc.os-images v1.15.0 [1].

[1] https://github.com/stackhpc/ansible-role-os-images/pull/63

Change-Id: I1a05d448031cb415cec4cd601a030abdb8a5eba5
(cherry picked from commit 0e4af118ebd03f697a7b460868772ee5f2dd0442)
